docs/devGuide/workflow.md

```diff
@@ -37,19 +37,19 @@ The sections below has more information about various stages of submitting a PR.
   Once done, see these screenshots for sample debugging configurations, and tweak them further to suit your needs.
   
   **1. Using the docs as a development environment**, with:
-    - the [lazy reload `-o`](https://markbind.org/userGuide/cliCommands.html#serve-command) option to speed up page building
-    - the `-d` developer option. (see [below](#editing-frontend-features))
-    
+  * the [lazy reload `-o`](https://markbind.org/userGuide/cliCommands.html#serve-command) option to speed up page building
```

Comment:
I think that the reason why this is hardcoded in is because of the separation of the developer guide and user guide websites? When I tweak the settings in site.json to match those of the developer guide, and change the link to a relative one, the page won't be found.

Reply:
Ahh right. Forgot that on the main site the developer docs has a different base URL. I guess we can leave this as it is for now. We can maybe think of a better way to structure the 2 sites in a future PR :)